code/modules/antagonists/wizard/grand_ritual/grand_ritual_finale.dm

```diff
@@ -399,6 +400,57 @@
 			mode.execute_roundstart_rule(meteors) // Meteors will continue until morale is crushed.
 			priority_announce("Meteors have been detected on collision course with the station.", "Meteor Alert", ANNOUNCER_METEORS)
 
+/**
+ * Gives the wizard a defensive/mood buff and a Wabbajack, a juiced up chaos staff that will surely break something.
+ * Everyone but the wizard goes crazy, suffers major brain damage, and is given a vendetta against the wizard.
+ * Already insane people are instead cured of their madness, ignoring any other effects as the station around them loses its marbles.
+ */
+/datum/grand_finale/cheese
+	/**we don't set name, desc and others, thus we won't appear in the radial choice of a normal finale rune */
```

Comment:
```suggestion
	// we don't set name, desc and others, thus we won't appear in the radial choice of a normal finale rune
```
Using the `/** */` format will probably mess up the doc comments for the following proc